graphistry/pygraphistry.py

```diff
@@ -242,12 +245,63 @@ def sso_login(org_name=None, idp_name=None, sso_timeout=SSO_GET_TOKEN_ELAPSE_SEC
         except Exception:  # required to log on
             # print("required to log on")
             PyGraphistry.sso_state(arrow_uploader.sso_state)
-
             auth_url = arrow_uploader.sso_auth_url
             # print("auth_url : {}".format(auth_url))
             if auth_url and not PyGraphistry.api_token():
                 PyGraphistry._handle_auth_url(auth_url, sso_timeout, sso_opt_into_type)
-                return auth_url
+  
+    @staticmethod
+    def databricks_notebook_sso_login( 
```

Comment:
is this for databricks notebooks or databricks dashboards?

Where in dashboards, I think the idea is first cell is a non-blocking print, and second cell is a blocking poll on successful sso

Where in notebooks... ?